src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./blech32";
+export { Blech32Address } from "./blech32Addr";
```

Comment:
are you sure this works? I recall we may need to export all

Reply:
it seems to work yes